### PR TITLE
net-libs/libtirpc: bump to EAPI=6

### PR DESCRIPTION
**Pull request by candrews**
Package-Manager: Portage-2.3.6, Repoman-2.3.2

### CODE REVIEW

File: net-libs/libtirpc/libtirpc-1.0.1-r3.ebuild [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+multilib_src_install_all() {
+	einstalldocs
+
+	insinto /etc
+	doins doc/netconfig
+
+	insinto /usr/include/tirpc
+	doins -r "${WORKDIR}"/tirpc/*
```
Comment (SoapGentoo) [inline net-libs/libtirpc/libtirpc-1.0.1-r3.ebuild]: `doins -r "${WORKDIR}"/tirpc/.` (always avoid unnecessary globs)

File: net-libs/libtirpc/libtirpc-1.0.1-r3.ebuild [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+	# makes sure that the linking order for nfs-utils is proper, as
+	# libtool would inject a libgssglue dependency in the list.
+	use static-libs || prune_libtool_files
```
Comment (SoapGentoo) [inline net-libs/libtirpc/libtirpc-1.0.1-r3.ebuild]: 1. use the find command instead, `eutils` stuff sucks (https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives)
2. check if the package installs `.pc` files. If so, nix the `.la` files unconditionally

File: net-libs/libtirpc/libtirpc-1.0.1-r3.ebuild [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+	"${FILESDIR}/${P}_ifdef-out-yp-headers-742bbdff6ddf.patch"
+	"${FILESDIR}/${P}_remove-nis-h-dep-5f00f8c78c5d.patch"
+	"${FILESDIR}/${P}_add-des_impl-c-7f6bb9a3467a.patch"
+	"${FILESDIR}/${P}_remove-des-deps-to-glibc-503ac2e9fa56.patch"
+	"${FILESDIR}/${P}_uclibc-use-memset-not-bzero.patch"
+)
+
+src_prepare() {
+	default
+	cp -r "${WORKDIR}"/tirpc "${S}"/ || die
```
Comment (SoapGentoo) [inline net-libs/libtirpc/libtirpc-1.0.1-r3.ebuild]: `cp -r "${WORKDIR}"/tirpc . || die`

File: net-libs/libtirpc/libtirpc-1.0.1-r3.ebuild [hunk 4]
```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
```
Comment (SoapGentoo) [inline net-libs/libtirpc/libtirpc-1.0.1-r3.ebuild]: @Polynomial-C won't like this...

File: net-libs/libtirpc/libtirpc-1.0.1-r3.ebuild (hunk 2 above)
Comment (candrews) [inline net-libs/libtirpc/libtirpc-1.0.1-r3.ebuild]: 👍

File: net-libs/libtirpc/libtirpc-1.0.1-r3.ebuild (hunk 1 above)
Comment (candrews) [inline net-libs/libtirpc/libtirpc-1.0.1-r3.ebuild]: 👍

File: net-libs/libtirpc/libtirpc-1.0.1-r3.ebuild (hunk 3 above)
Comment (candrews) [inline net-libs/libtirpc/libtirpc-1.0.1-r3.ebuild]: done

**Pull request closed without merging**